setup.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 2.1.3
+current_version = 2.1.4
```

Comment:
For feature PRs: There's no need to manually update the version. It will be automatically bumped using the make bump command.